src/api/me.py

```diff
@@ -834,10 +832,8 @@ async def update_review(
         await ReviewDatabaseHandler.update_alcohol_rating(db.alcohols, alcohol_id, rating, review_update_payload.rating)
     await ReviewDatabaseHandler.update_user_rating(db.users, current_user['_id'], rating, review_update_payload.rating)
 
-    if review_update_payload.review:
-        await check_review(
-            settings.HATE_SPEECH_DETECTION_SERVICE_URL, review_update_payload.review, db.reviews, review_id
-        )
+    if review_update_payload.review and client.check_review(review_update_payload.review):
```

Comment:
Jakie są odpowiedzi w przypadku gdy klient wykryje i nie wykryje obraźliwy język?
Bo tutaj jeżeli będzie jakakolwiek odpowiedź to podbijasz w bazie ilość zgłoszeń

Reply:
Zwraca tylko True/False

Resztę powinien ogarnąć try i zwrócić null